content/archive/ARCHIVED; User State is State of the Art.md

```
@@ -25,9 +26,7 @@ While we perceive psychological continuity across contexts and time, closer insp
 In short, it's messy. Or, rather, elegant emergent complexity. 
 
 Each human self isn't just one mythical [Ship of Theseus](https://en.wikipedia.org/wiki/Ship_of_Theseus)--planks being replaced one by one over slow years--but a fleet of them, all with full, manual and autonomous CRUD operations.
-
-## Digital Twins Are Naïve
-
+# Digital Twins Are Naïve
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)